src/components/application_manager/include/application_manager/commands/hmi/navi_is_ready_request.h

```diff
@@ -55,12 +56,17 @@ class NaviIsReadyRequest : public RequestToHMI {
   /**
    * @brief NaviIsReadyRequest class destructor
    **/
-  virtual ~NaviIsReadyRequest();
+  ~NaviIsReadyRequest() OVERRIDE;
```

Comment:
@dtrunov What is the purpose of `OVERRIDE` macros in destructor? The `C++11` keyword `override` is not applicable for destructors. Previous definition makes sense in terms of current class inheritance.